api/src/main/java/org/loomdev/api/util/ChatColor.java

```diff
@@ -5,6 +5,7 @@
 /**
  * Chat color utilities.
  */
+// TODO replace with net.kyori.adventure.text.format.NamedTextColor
```

Comment:
You are right about the static fields these could be replaced by using NamedTextColor. (didn't know that existed 🙃) The reason this class was initially made was the translate method, which we have not been able to find in adventure. So we would at least like to keep that 😄